website/docs/docs/dbt-cloud-apis/sl-python-sdk.md

```diff
@@ -0,0 +1,138 @@
+
+<Tabs>
+<TabItem value="sync" label="Sync installation">
+
+Sync installation means your program waits for each task to finish before moving on to the next one. 
+
+It's simpler and easier to understand. Suitable for smaller tasks or when your program doesn't need to handle many tasks at the same time.
+
+```bash
+pip install dbt-sl-sdk[sync]
+```
+</TabItem>
+
+<TabItem value="async" label="Async installation">
+
+Async installation means your program can start a task and then move on to other tasks while waiting for the first one to finish.
+
+It can handle many tasks at once without waiting, making it faster and more efficient for larger tasks or when you need to manage multiple tasks at the same time.
+
+```bash
+pip install dbt-sl-sdk[async]
+```
+</TabItem>
+</Tabs>
+
+## Usage
+To run operations against the Semantic Layer APIs, instantiate (create an instance of) a `SemanticLayerClient` with your specific [API connection parameters](/docs/dbt-cloud-apis/sl-api-overview):
+
+```python
+from dbtsl import SemanticLayerClient
+
+client = SemanticLayerClient(
+    environment_id=123,
+    auth_token="<your-semantic-layer-api-token>",
+    host="semantic-layer.cloud.getdbt.com",
+)
+
+# query the first metric by `metric_time`
+def main():
+    with client.session():
+        metrics = client.metrics()
+        table = client.query(
+            metrics=[metrics[0].name],
+            group_by=["metric_time"],
+        )
+        print(table)
+
+main()
+```
+
+**Note**: All method calls that reach out to the APIs need to be within a `client.session()` context manager. This allows the client to establish a connection to the APIs only once and reuse the same connection between API calls.
+
+### asyncio usage
+If you're using asyncio, import `AsyncSemanticLayerClient` from `dbtsl.asyncio`. The APIs of `SemanticLayerClient` and `AsyncSemanticLayerClient` are identical, but the async version has async methods that you need to `await`.
+
+```python
+import asyncio
+from dbtsl.asyncio import AsyncSemanticLayerClient
+
+client = AsyncSemanticLayerClient(
+    environment_id=123,
+    auth_token="<your-semantic-layer-api-token>",
+    host="semantic-layer.cloud.getdbt.com",
+)
+
+async def main():
+    async with client.session():
+        metrics = await client.metrics()
+        table = await client.query(
+            metrics=[metrics[0].name],
+            group_by=["metric_time"],
+        )
+        print(table)
+
+asyncio.run(main())
+
+```
+
+## Integrate with dataframe libraries
+
+The SDK returns all query data as [pyarrow](https://arrow.apache.org/docs/python/index.html) tables. 
+
+To use the data with libraries like [pandas](https://pandas.pydata.org/) or [polars](https://pola.rs/), manually convert the data into the desired format:
```

Comment:
Also note that the library doesn't come bundled with Polars or Pandas since we don't wanna force users to install either. If they're using it, they need to add them as a dependency of their project.